lib/presentation/pages/add_expenses_page/add_expenses_page.dart

```diff
@@ -18,7 +25,45 @@ import 'package:trip_tally/presentation/widgets/suffix_icon_text_field.dart';
 
 @RoutePage()
 class AddExpensesPage extends StatelessWidget {
-  const AddExpensesPage({super.key});
+  const AddExpensesPage({
+    super.key,
+    @visibleForTesting this.bloc,
+  });
+
+  final AddExpensesBloc? bloc;
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => bloc ?? getIt<AddExpensesBloc>(),
+      child: BlocConsumer<AddExpensesBloc, AddExpensesState>(
+        listener: (context, state) => state.maybeWhen(
+          orElse: () => const SizedBox.shrink(),
+          success: () => customSnackBar(context, 'Success', color: AppColors.wePeep),
+        ),
+        builder: (context, state) => state.maybeWhen(
+          orElse: SizedBox.shrink,
+          success: _Body.new,
+          initial: _Body.new,
```

Comment:
what will happen on loading state?
orElse should not be a shrink coz it will look misleading for the user